src/swap/v1_1/index.ts

```diff
@@ -229,7 +228,10 @@ function getFixedInputSwapQuote({
   decimals: {assetIn: number; assetOut: number};
 }): SwapQuote {
   if (pool.status !== PoolStatus.READY) {
-    throw new TinymanError({pool, assetIn}, "Trying to swap on a non-existent pool");
+    throw new SwapQuoteError(
+      SwapQuoteErrorType.NoAvailablePoolError,
+      "Trying to swap on a non-existent pool"
```

Comment:
I think we can make the default message better
```suggestion
      "There is not an available pool for this asset pair"
```